src/store/transactions/actions.ts

```diff
@@ -4,29 +4,32 @@ import { Contract } from '@models/contract.interface';
 import { fetchTx } from '@common/api/transactions';
 import { queryWith0x } from '@common/utils';
 import { fetchContract } from '@store/contracts';
+import { selectCurrentNetworkUrl } from '@store/ui/selectors';
 
 export const fetchTransaction = createAsyncThunk<Transaction[], string>(
   'transaction/fetch',
-  async (query, { dispatch, rejectWithValue }) => {
+  async (query, { dispatch, rejectWithValue, getState }) => {
+    // @ts-ignore
+    const apiServer = selectCurrentNetworkUrl(getState());
```

Comment:
```suggestion
    const apiServer = selectCurrentNetworkUrl(getState() as any) as string;
```

This isn't so different, but think it's preferable as at least we're being very explicit about what we're not typing.